index_tags.md

```diff
@@ -3,46 +3,17 @@
 
 # Introduction
 
-Matroska aims to become THE standard of multimedia container formats. It was derived from a project called [MCF](http://mukoli.free.fr/mcf/mcf.html), but differentiates from it significantly because it is based on [EBML](https://github.com/Matroska-Org/ebml-specification/blob/master/specification.markdown) (Extensible Binary Meta Language), a binary derivative of XML. EBML enables significant advantages in terms of future format extensibility, without breaking file support in old parsers.
+Matroska aims to become THE standard of multimedia container formats. It can store timestamped multimedia data but also chapters and tags. The tags add important metadata to indentify and classify the information found in a Matroska Segment. It can tag a whole Segment, separate Tracks, individual Chapters or Attachments.
```

Comment:
Either backtick element names "Segment", "Tracks", "Chapters", "Attachments", or write them in lower-case.